src/Input/Input.tsx

```diff
@@ -89,29 +89,42 @@ export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
    * @deprecated
    */
   static?: boolean;
+  /** Quando attivo rimuove il componente contenitore dell'Input. Utile per un controllo maggiore dello styling */
+  noWrapper?: boolean;
+  /** Funzione callback per quando il campo riceve il focus */
+  onFocus?: React.FocusEventHandler<HTMLInputElement>;
+  /** Funzione callback per quando il campo perde il focus */
+  onBlur?: React.FocusEventHandler<HTMLInputElement>;
```

Comment:
These should be already inherited by `InputHTMLAttributes<HTMLInputElement>`, so no need to redefined them again.

Reply:
done!